src/Frontend/index.tsx

```diff
@@ -13,6 +16,11 @@ export default function Initial() {
         <Stack.Screen name="LandingPage" component={LandingPage} />
         <Stack.Screen name="LogIn" component={LogInPage} />
         <Stack.Screen name="SignUp" component={SignUpPage} />
+        <Stack.Screen name="DashBoard" component={Tabs} />
+        <Stack.Screen name="Free-Food" component={Food} />
+        <Stack.Screen name="Free Non-Food" component={NonFood} />
```

Comment:
Page should be Add, in Add page there should be a check whether it is a Non-Food or Food
for example

```
const [currentPage, setCurrentPage] = useState<PageState>(PageState.VIEW);
  return (
    <>
      {currentPage === PageState.VIEW && (
        <ViewContactPage
          isUserManager={isUserManager}
          organizationId={organizationId}
          onAddOrEditContact={handleAddOrEditContact}
          onEditContact={handleEditContact}
        />
      )}
      {(currentPage === PageState.ADD || currentPage === PageState.EDIT) && (
        <AddOrEditContactPage
          isUserManager={isUserManager}
          onBack={handleBack}
          editingContact={
            currentPage === PageState.EDIT ? editingContact : null
          }
        />
      )}
    </>
  );
};
```

Reply:
Just an example PageState.VIEW is an enum. In our case there will be AddOption.FOOD or AddOption.NONFOOD

```
export enum PageState {
  VIEW = 'view',
  ADD = 'add',
  EDIT = 'edit',
}
```